lua/hbac/telescope/make_display.lua

```diff
@@ -0,0 +1,64 @@
+local hbac_config = require("hbac.setup").opts
+local state = require("hbac.state")
+
+local entry_display = require("telescope.pickers.entry_display")
+local has_devicons, devicons = pcall(require, "nvim-web-devicons")
```

Comment:
since we require telescope, devicons and plenary, all of this should be mentioned in the README